Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -2,9 +2,11 @@
 
 #pragma once
 
+#include "PSTypes.h"
 #include "Subsystems/WorldSubsystem.h"
 #include "PSWorldSubsystem.generated.h"
 
+DECLARE_MULTICAST_DELEGATE_OneParam(FCurrentRowDataChanged, FPSRowData);
```

Comment:
Please move DECLARE macro from global space to local right above delegate property

Reply:
done